### PR TITLE
Increase the interval for Zabbix checks to save on both DB load and space

### PR DESCRIPTION
**Pull request by rharrison10**

### CODE REVIEW

**Comment by blentz** [comment]
```
   v2 = 349.93 new values per second.
   v3 = 1,583.2 new values per second.

   v2 - 204.455341506 items per host.
   v3 - 236.252475248 items per host.

   v2 - 571 enabled hosts
   v3 - 388 enabled hosts

   v2 - 116,749 enabled items
   v3 - 96,609 enabled items

   v2 - 0.4402 items per second per host.
   v3 - 3.916584158 items per second per host.
```

So, v3 zabbix has fewer hosts, comparable numbers of items-per-host, but is tracking more items-per-second.

Also, to provide some disk-based metrics, we'll look at two examples.
- 300 second interval
- 7 days of history
- 365 days of trends

versus
- 60 second check interval  
- 90 days of history       (exact values stored based on interval)
- 365 days of trends     (hourly min/max/average stored)

For scenario 2, Zabbix stores 129,600 values for the 90 days of history, per item,
per host and 19,800 values for the trends, per item, per host.

Scenario 1 stores 2,016 values for the 7 days of history, per item, per
host and 25,776 values for the trends, per item, per host.

That's a savings of 121,608 values per item, per host for a one-year data set.

A regular INT in MySQL is 4 bytes. So, we're talking about a 475kib savings on 1 year's worth of data per item per host. Multiplied out to v2 scale, we've saved ourselves at least 55Gib by doing this, not including the ability to run a smaller RDS instance which is also a cost savings above and beyond just reducing the performance requirements.

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 1]
```diff
@@ -82,15 +82,15 @@ zagg_client_config:
 # Generic Linux Checks
 host_monitoring_cron:
 - name: send pcp ping every 5 mintues
-  minute: "*/5"
+  minute: "*/15"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Decision: Leave at 5

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 2]
```diff
@@ -82,15 +82,15 @@ zagg_client_config:
...
   job: ops-runner -f -s 60 -n cspp.pcp.ping /usr/bin/cron-send-pcp-ping
 
-- name: run pcp checks every minute
-  minute: "*"
+- name: run pcp checks every 5 minutes
+  minute: "*/5"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Add comment:

```
# This is needed every minute for the performance team
```

Decision: Leave at 1 minute

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 3]
```diff
@@ -82,15 +82,15 @@ zagg_client_config:
...
   job: ops-runner -f -s 15 -n ozpc.send.pcp /usr/bin/ops-zagg-pcp-client
 
-- name: run pcp sampler every minute
-  minute: "*"
+- name: run pcp sampler every 5 minutes
+  minute: "*/5"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Add comment:

```
# This is needed every minute for the performance team
```

Decision: Leave at 1 minute

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 4]
```diff
@@ -99,56 +99,56 @@ host_monitoring_cron:
   job: ops-runner -f -s 300 -n ozc.send.heartbeat.full /usr/bin/ops-zagg-client --send-heartbeat
 
 - name: Do a quick heartbeat
-  minute: "*/5"
+  minute: "*/15"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Add comment:

```
# This is our heart beater and lets us know when a box is down. Too much time between checks means it'll take longer for us to know a box is down. See heartbeat trigger for details.
```

Decision: Leave at 5 minutes.

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 5]
```diff
@@ -99,56 +99,56 @@ host_monitoring_cron:
...
   job: ops-runner -f -s 60 -n ozc.send.heartbeat.quick /usr/bin/ops-zagg-client -k heartbeat.ping -o 1
 
 - name: run filesystem space checks every 10 minutes
   minute: "*/10"
   job: ops-runner -f -s 120 -n csfm.filesys.full /usr/bin/cron-send-filesystem-metrics
 
-- name: run disk TPS checks every minute
-  minute: "*"
+- name: run disk TPS checks every 5 minutes
+  minute: "*/5"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Decision: Leave at 1 minute, change history retention to 7 days.

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 6]
```diff
@@ -99,56 +99,56 @@ host_monitoring_cron:
...
   job: ops-runner -f -s 15 -n csdim.disk.tps /usr/bin/cron-send-disk-metrics
 
-- name: run network checks every minute
-  minute: "*"
+- name: run network checks every 5 minutes
+  minute: "*/5"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2:119]: Decision: ok with this

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 7]
```diff
@@ -99,56 +99,56 @@ host_monitoring_cron:
...
   job: ops-runner -f -s 15 -n csnm.network.int /usr/bin/cron-send-network-metrics
 
 # We might want to break docker checks out at some point.
-- name: run docker storage space checks every 10 minutes
-  minute: "*/10"
+- name: run docker storage space checks every 15 minutes
+  minute: "*/15"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2:124]: Decision: ok

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 8]
```diff
@@ -99,56 +99,56 @@ host_monitoring_cron:
...
   job: ops-runner -f -s 120 -n csdm.docker.storage /usr/bin/cron-send-docker-metrics
 
 - name: run docker info timer
-  minute: "*/10"
+  minute: "*/15"
   job: ops-runner -f -s 120 -n csdt.docker.timer /usr/bin/cron-send-docker-timer
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2:131]: Add comment:

```
# This check shows us when docker is in a bad state (either locked up or too slow to respond).
```

Decision: Leave at 10 minutes

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 9]
```diff
@@ -99,56 +99,56 @@ host_monitoring_cron:
...
 
 {% if osohm_host_type != 'ansible-tower' %}
 - name: run docker dns test
-  minute: "*/5"
+  minute: "*/10"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Add comment:

```
# This is an active issue, we need it to run on a frequent basis.
```

Decision: leave for now

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 10]
```diff
@@ -99,56 +99,56 @@ host_monitoring_cron:
...
   job: ops-runner -f -s 60 -n csddr.docker.dns.resolution /usr/bin/cron-send-docker-dns-resolution
 {% endif %}
 
 - name: run docker dns test on existing containers
-  minute: "*/5"
+  minute: "*/10"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Add comment:

```
# This is an active issue, we need it to run on a frequent basis.
```

Decision: leave for now

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 11]
```diff
@@ -110,45 +115,49 @@ host_monitoring_cron:
...
+# This is an active issue, we need it to run on a frequent basis.
 - name: run docker dns test on existing containers
   minute: "*/5"
   job: ops-runner -f -s 60 -n csdedr.docker.existing.dns.resolution /usr/bin/cron-send-docker-existing-dns-resolution
 
 {% if osohm_host_type == 'node' and osohm_monitor_dnsmasq|bool %}
 {# dnsmasq checks #}
 - name: send dnsmasq process count
-  minute: "*/2"
+  minute: "*/5"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2:148]: Decision: ok with this change

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 12]
```diff
@@ -110,45 +115,49 @@ host_monitoring_cron:
...
   job: ops-runner -f -s 15 -n cspc.openshift.dnsmasq.process.count /usr/bin/cron-send-process-count '^dnsmasq' openshift.dnsmasq.process.count
 
-- name: send openshift-node dnsmasq status every 4 minutes
-  minute: "*/4"
+- name: send openshift-node dnsmasq status every 10 minutes
+  minute: "*/15"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Decision: 5 minutes, fix comment

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 13]
```diff
@@ -110,45 +115,49 @@ host_monitoring_cron:
...
   job: ops-runner -f -s 50 -n csosms.openshift.dnsmasq /usr/bin/cron-send-os-dnsmasq-checks
 {# end if dnsmasq #}
 {% endif %}
 
 {% if osohm_host_type == 'master' or osohm_host_type == 'node' %}
 {# Openshift node checks #}
 - name: send openshift-node process count
-  minute: "*/2"
+  minute: "*/5"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2:160]: Decision: ok with this change

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 14]
```diff
@@ -160,11 +169,11 @@ host_monitoring_cron:
   job: ops-runner -f -s 60 -n cfor.openshift.node.ovs.stray.rules /usr/bin/cron-fix-ovs-rules
 
 - name: send cluster docker registry checks
-  minute: "*/2"
+  minute: "*/5"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Decision: leave at 2 minutes

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 15]
```diff
@@ -160,11 +169,11 @@ host_monitoring_cron:
...
   job: ops-runner -f -s 15 -n csrc.openshift.node.registry.checks /usr/bin/cron-send-registry-checks
 
 - name: send openshift-node cpu mem per process stats
-  minute: "*/5"
+  minute: "*/10"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Decision: leave at 5 minutes

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 16]
```diff
@@ -190,8 +199,8 @@ host_monitoring_cron:
 
 {% if osohm_host_type == 'master' %}
 {# Openshift Master checks #}
-- name: run create app every 5 minutes
-  minute: "*/5"
+- name: run create app every 15 minutes
+  minute: "*/15"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Decision: leave at 5 minutes

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 17]
```diff
@@ -200,13 +209,13 @@ host_monitoring_cron:
 
 {% if osohm_master_ha|bool %}
 - name: send openshift-master process count
-  minute: "*/2"
+  minute: "*/5"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2:212]: Decision: ok with this change

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 18]
```diff
@@ -200,13 +209,13 @@ host_monitoring_cron:
...
   job: ops-runner -f -s 15 -n cspc.openshift.master.process.count /usr/bin/cron-send-process-count '^/usr/bin/openshift start master controllers' openshift.master.process.count
 {% endif %}
 
 {% if not osohm_master_ha|bool %}
 - name: send openshift-master process count
-  minute: "*/2"
+  minute: "*/5"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2:218]: Decision: ok with this change

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 19]
```diff
@@ -215,20 +224,20 @@ host_monitoring_cron:
   minute: "0"
   job: ops-runner -f -s 15 -n csosmm.openshift.master.counts /usr/bin/cron-send-os-master-metrics --project-count --pod-count --user-count --pv-info
 
-- name: send openshift-master /healthz status every 4 minutes
-  minute: "*/4"
+- name: send openshift-master /healthz status every 5 minutes
+  minute: "*/5"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2:228]: Decision: ok with this

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 20]
```diff
@@ -215,20 +224,20 @@ host_monitoring_cron:
...
   job: ops-runner -f -s 15 -n csosmm.openshift.master.api.healthz /usr/bin/cron-send-os-master-metrics --healthz --api-ping --metrics --node-checks -v &>> /var/log/csosmm-notready.log
 
 - name: send openshift-master local (test https://127.0.0.1) status every 5 minutes
   minute: "*/5"
   job: ops-runner -f -s 60 -n csosmm.openshift.master.api.healthz /usr/bin/cron-send-os-master-metrics --local
 
-- name: send openshift-master SkyDNS status every 4 minutes
-  minute: "*/4"
+- name: send openshift-master SkyDNS status every 10 minutes
+  minute: "*/10"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Decision: Critical service, change to 5 minutes

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 21]
```diff
@@ -215,20 +224,20 @@ host_monitoring_cron:
...
   job: ops-runner -f -s 50 -n csosms.openshift.master.skydns /usr/bin/cron-send-os-skydns-checks
 
 - name: send openshift-master etcd status
-  minute: "*/2"
+  minute: "*/5"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Decision: critical service, leave at 2 minutes.

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 22]
```diff
@@ -248,20 +257,20 @@ host_monitoring_cron:
   job: ops-runner -f -s 60 -n cscc.openshift.master.api.connections.443 /usr/bin/cron-send-connection-count openshift 443 openshift.master.api.port_443.connections.established
 
 - name: send openshift master api cpu mem per process stats
-  minute: "*/5"
+  minute: "*/10"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Decision: leave at 5 minutes

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 23]
```diff
@@ -248,20 +257,20 @@ host_monitoring_cron:
...
   job: ops-runner -f -s 60 -n cscpm.openshift.master.api.process.cpu.mem.stats /usr/bin/cron-send-cpu-mem-stats 'openshift start master api' openshift.master.api.process
 
 - name: send openshift master controllers cpu mem per process stats
-  minute: "*/5"
+  minute: "*/10"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Decision: leave at 5 minutes

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 24]
```diff
@@ -248,20 +257,20 @@ host_monitoring_cron:
...
   job: ops-runner -f -s 60 -n cscpm.openshift.master.controllers.process.cpu.mem.stats /usr/bin/cron-send-cpu-mem-stats 'openshift start master controllers' openshift.master.controllers.process
 
 - name: send etcd cpu mem per process stats
-  minute: "*/5"
+  minute: "*/10"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Decision: leave at 5 minutes

File: ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2 [hunk 25]
```diff
@@ -248,20 +257,20 @@ host_monitoring_cron:
...
   job: ops-runner -f -s 60 -n cscpm.openshift.etcd.process.cpu.mem.stats /usr/bin/cron-send-cpu-mem-stats etcd openshift.etcd.process
 
 - name: "OpenShift event watcher"
   hour: "*"
-  minute: "*/2"
+  minute: "*/10"
```
Comment (twiest) [inline ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2]: Decision: leave at 2 minutes

**Comment by twiest** [comment]
:+1:

**Pull request merged**